src/main/webapp/script.js

```diff
@@ -1,63 +1,77 @@
-let zoomedIn = false;
-let aqLayerOn = true;
-let routeVisible = false;
-
-function changeBackgroundImage(imagePath) {
-    document.body.style.backgroundImage = `url('${imagePath}')`;
+let map;
+let aqLayer;
+
+function aqLayerControl(controlDiv) {
+    // Set CSS for the control border.
```

Comment:
Can these properties be controlled via a class, and specified in the style.css instead?

Reply:
Yep, done